tams_pr2_guzheng/ActionParameters parameters

# target string to pluck
string string

# finger to use
# one of ff/mf/rf/th
string finger 
